bc_ccv_rstudio_singularity/template/bin/auth

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Confirm username is supplied
+if [[ $# -lt 1 ]]; then
+  echo "Usage: auth USERNAME"
+  exit 1
+fi
+USERNAME="${1}"
+
+# Confirm password environment variable exists
+if [[ -z ${RSTUDIO_PASSWORD} ]]; then
+  echo "The environment variable RSTUDIO_PASSWORD is not set"
+  exit 1
+fi
+
+# Read in the password from user
+read -s -p "Password: " PASSWORD
```

Comment:
Is this an actual password? Or, is this a dummy or placeholder?

Reply:
These are dummy passwords